llvm-project/stage2.cmake

```diff
@@ -0,0 +1,74 @@
+# Please try to keep these cmake variables alphabetically sorted.
+
+# Enable optimizations, as opposed to a debug build.
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "")
+
+# Use stage1's clang to bootstrap stage2, as opposed to Alpine's clang (which
+# shouldn't be installed on the system at this point, but would be installed to
+# /usr/bin/clang).
+set(CMAKE_CXX_COMPILER "/usr/local/bin/clang++" CACHE FILEPATH "")
+
+# Use libc++ from stage1.
+# We could not do this in stage1 because Alpine doesn't distribue libc++ ATM.
+# TODO: does clang cmake not have a CLANG_DEFAULT_* flag for setting this?
```

Comment:
regardless, stage1 could not default to libc++ because stage1 clang is built before libc++, and alpine does not distribute libc++.

Reply:
Looks like there is CLANG_DEFAULT_RTLIB.
https://reviews.llvm.org/D22663